### ISSUE TITLE
More portable Galago Pro, 13 inches and lighter

### ISSUE THREAD

**Issue by spastorino** [system76/laptop-suggestions#97]
A more portable and light computer would be desirable and one that it's really 13 inches size, not only the screen but the whole computer. Current one has some huge borders. So everything that is reducing it's size and weight would be great.

**Comment by netgifted** [comment]
Yes...I think I repeated this request but I suggest an ARM based processor for an ultra light System76

**Comment by jonpolak** [comment]
I want 16:10 or 3:2 so it is better for portrait documents. High end screen please with no PWM !!
